js/script.js

```diff
@@ -0,0 +1,185 @@
+let allTasks = [];
+let valueInput = '';
+let input = null;
+let content;
+let activeEditTask = null;
+
+const buttonTasksAdd = async () => {
+  if (!valueInput.trim()) { return; }
+
+  allTasks.push({
+    text: valueInput.trim(),
+    isCheck: false,
+  });
+
+  const resp = await fetch('http://localhost:8000/createTask', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Origin': '*'
+    },
+    body: JSON.stringify({
+      text: valueInput.trim(),
+      isCheck: false,
+    }),
+  });
+  let result = await resp.json();
+  allTasks  = result.data;
+  valueInput = '';
+  input.value = '';
+  render();
+}
+
+const newLocal = 'change';
+
+window.onload = async function init() {
+  input = document.getElementById('add-task');
+  input.addEventListener('change', updateValue);
+  content = document.getElementById('content-page');
+  const button = document.querySelector('.add__btn');
+  button.addEventListener('click', buttonTasksAdd);
+
+  const resp = await fetch('http://localhost:8000/allTasks', {
+    method: "GET"
+  });
+  let result = await resp.json();
+  allTasks  = result.data;
+  render();
+}
+
+const updateValue = (event) => {
+  valueInput = event.target.value;
+}
+
+const render = () => {
+
+  while (content && content.firstChild) {
+    content.removeChild(content.firstChild);
+  };
+
+  allTasks.sort((a, b) => a.isCheck > b.isCheck ? 1 : a.isCheck < b.isCheck ? -1: 0);

```

Comment:
sorting mutates the array. It shouldn't be like that. Copy it to another array and work with it. In general, it is better to do sorting on backend and already get a sorted array